Content.Shared/Teleportation/Systems/SharedPortalSystem.cs

```diff
@@ -18,10 +18,11 @@ namespace Content.Shared.Teleportation.Systems;
 /// </summary>
 public abstract class SharedPortalSystem : EntitySystem
 {
-    [Dependency] private readonly IRobustRandom _random = default!;
-    [Dependency] private readonly INetManager _netMan = default!;
     [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly INetManager _netMan = default!;
     [Dependency] private readonly SharedPullingSystem _pulling = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly SharedTransformSystem _transform = default!;
```

Comment:
Why re-organise these to a random order?

Reply:
alphabetic order as opposed to random

Reply:
(on field name not type name)

Reply:
Better sort by type, i guess